internal/services/storage/registration.go

```diff
@@ -29,7 +29,7 @@ func (r Registration) SupportedDataSources() map[string]*pluginsdk.Resource {
 func (r Registration) SupportedResources() map[string]*pluginsdk.Resource {
 	return map[string]*pluginsdk.Resource{
 		"azurestack_storage_account":   storageAccount(),
-		"azurestack_storage_blob":      storageBlob(),
+		"azurestack_storage_blob":      resourceStorageBlob(),
```

Comment:
could we make this consistant?
```suggestion
		"azurestack_storage_blob":      storageBlob(),
```

all function names should have resource removed like this

Reply:
done :D